packages/block-library/src/navigation/use-navigation-menu.js

```diff
@@ -79,6 +79,12 @@ export default function useNavigationMenu( ref ) {
 					'canUser',
 					[ 'delete', 'navigation', ref ]
 				),
+				canUserCreateNavigation:
+					canUser( 'create', 'navigation' ) || undefined,
```

Comment:
Why `|| undefined`?

Reply:
I don't think we need it. Probably some kind of hangover from copy/paste. 

Fixed.